src/test/resources/org/powerflows/dmn/io/yaml/test_multiple_decisions.yml

```diff
@@ -0,0 +1,109 @@
+id: sample_decision_1
```

Comment:
Is it required to keep in org/powerflows/dmn/io/yaml/ ? Or is it temporary solution?